src/concepts/html/README.md

```diff
@@ -2,15 +2,24 @@
 description: The procedural macro for generating HTML and SVG
 ---
 
-# Using html!
+# Using `html!`
 
 The `html!` macro allows you to write HTML and SVG code declaratively. It is similar to JSX \(a Javascript extension which allows you to write HTML code inside of Javascript\).
 
 **Important notes**
 
 1. The `html!` macro only accepts one root html node \(you can counteract this by [using fragments or iterators](lists.md)\)
 2. An empty `html! {}` invocation is valid and will not render anything
-3. Literals must always be quoted and wrapped in braces: `html! { "Hello, World" }`
+3. Literals in element content must always be quoted and wrapped in braces
```

Comment:
Not sure if "in element content" helps here?

Reply:
Any better suggestions? I thought a bit but that didn't help ;-)

Reply:
Could we just:
```suggestion
3. Literals must always be quoted and wrapped in braces
```

Reply:
That was what confused me because literal values passed to attributes do not need braces.

Reply:
Oh right. That makes sense. I guess 
```suggestion
3. Literals inside of tags must always be quoted and wrapped in braces (e.g. `<p>{"hello world"}</p>`)
```